ui/panels-plugin/src/plugins/time-series-chart/TimeSeriesChartPanel.test.tsx

```diff
@@ -77,6 +77,18 @@ const TEST_TIME_SERIES_PANEL: TimeSeriesChartProps = {
   },
 };
 
+function getLegendByName(name?: string) {
+  if (typeof name !== 'string') {
+    throw new Error('Legend name must be a string.');
+  }
```

Comment:
Making the name optional allows me to avoid some type pain farther below because typescript can't be sure a specific index of the series has content or not.